src/dune_rules/coq/coq_rules.ml

```diff
@@ -952,11 +952,14 @@ let coqtop_args_theory ~sctx ~dir ~dir_contents (s : Coq_stanza.Theory.t) coq_mo
 (* This is here for compatibility with Coq < 8.11, which expects plugin files to
    be in the folder containing the `.vo` files *)
 let coq_plugins_install_rules ~scope ~package ~dst_dir (s : Coq_stanza.Theory.t) =
-  let lib_db = Scope.libs scope in
   let+ ml_libs =
     (* get_libraries from Coq's ML dependencies *)
-    Resolve.Memo.read_memo
-      (Resolve.Memo.List.map ~f:(Lib.DB.resolve lib_db) s.buildable.plugins)
+    Resolve.Memo.List.map s.buildable.plugins ~f:(fun lib ->
```

Comment:
The point of this shuffling is that you will now no longer load the OCaml library database unless you use plugins. It's possibly unnecessary, but it seems like a good thing to have anyway.